new-docs/content/rust/reference/polar/classes.md

```diff
@@ -97,7 +97,7 @@ using a method, you may iterate over it with [the `in`
 operator](polar-syntax#in-list-membership) or destructure it with [pattern
 matching](polar-syntax#patterns-and-matching).
 
-### HashMaps
+### `HashMap`s
```

Comment:
"`HashMap`s" typesets poorly. I suggest "Hash Maps" as the section title, and the first line begin with "A Rust `HashMap` is mapped to a Polar dictionary, but require that the key be a `String`:`.

Reply:
made roughly the suggested change. Don't particularly want to fix now, but there are other spots where we have `backticked code` followed immediately by non-backticked text (usually pluralizing, as was the case here).